.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           command: sudo pip install awscli
       - run:
           name: Deploy to S3
-          command: aws s3 sync src/ s3://plot-viewer
+          command: aws s3 sync s3://plot-viewer
```

Comment:
Did you test this command?
I haven't, but it looks incorrect to me. Here are the docs: https://docs.aws.amazon.com/cli/latest/reference/s3/sync.html

The command signature is: `sync <LocalPath> <S3Uri> or <S3Uri> <LocalPath> or <S3Uri> <S3Uri>`

Reply:
I'm not sure how I can test without merging to master, I switched to 
"aws s3 sync / s3://plot-viewer" should that work?

Reply:
You definitely need to test these things out before merging them in. There's rarely things that are so CI dependent that you'd need to commit and push to test it out. There's almost always a way.

Copy and paste that command on your cmd line terminal and try it out!

If it complains about permissions, fix it and try it again. You also have permissions to create S3 buckets in AWS. Create one, install the awscli and test out the command with your own credentials.

Reply:
Done